src/dict.c

```diff
@@ -51,10 +51,15 @@
  * enable/disable resizing of the hash table as needed. This is very important
  * for Redis, as we use copy-on-write and don't want to move too much memory
  * around when there is a child performing saving operations.
+ * 使用 dictEnableResize() / dictDisableResize() 方法可以让我们很容易去控制哈希表是否进行大小再分配
+ * 因为 redis 使用的是写时复制，在做一些内存安全的操作的时候对内存做过多的操作(如新建或复制)是很消耗性能的
  *
  * Note that even when dict_can_resize is set to 0, not all resizes are
  * prevented: a hash table is still allowed to grow if the ratio between
- * the number of elements and the buckets > dict_force_resize_ratio. */
+ * the number of elements and the buckets > dict_force_resize_ratio.
+ * 需要注意的是，即使 dict_can_resize 置为 0 时，也不意味着所有 resize 行为都被禁止了
+ * 当哈希表元素与桶大小比值大于阈值 (dict_force_resize_ratio = 5), redis 还是会对哈希表进行 resize
+ */
```

Comment:
就不改动到上面一行英文注释，可以尽量按照新增的方式进行
```suggestion
 * the number of elements and the buckets > dict_force_resize_ratio. */
/* 需要注意的是，即使 dict_can_resize 置为 0 了，也不意味着所有 resize 行为都被禁止了
 * 当哈希表元素与桶大小比值大于阈值 (dict_force_resize_ratio = 5), redis 还是会对哈希表进行 resize */
```